tools/osbuilder/rootfs-builder/rootfs.sh

```diff
@@ -762,6 +770,44 @@ EOF
 	info "Create /etc/resolv.conf file in rootfs if not exist"
 	touch "$dns_file"
 
+    if [ -n "${AA_KBC}" ]; then
+		if [ "${AA_KBC}" == "offline_sev_kbc" ]; then
+			info "Adding agent config for ${AA_KBC}"
+			AA_KBC_PARAMS="offline_sev_kbc::null" envsubst < "${script_dir}/agent-config.toml.in" | tee "${ROOTFS_DIR}/etc/agent-config.toml"
+		fi
+		if [ "${AA_KBC}" == "online_sev_kbc" ]; then
+			info "Adding agent config for ${AA_KBC}"
+			#KBC URI will be specified in the config file via kernel params
+			AA_KBC_PARAMS="online_sev_kbc::123.123.123.123:44444" envsubst < "${script_dir}/agent-config.toml.in" | tee "${ROOTFS_DIR}/etc/agent-config.toml"
+		fi
```

Comment:
@fitzthum, @ryansavino - hey folks, I just wanted to check if SEV tests are still planning to use `offline_sev_kbc` and `online_sev_kbc` anymore, or have you switched to use cc_kbc?

Reply:
might be related. https://cloud-native.slack.com/archives/C039JSH0807/p1683139263612659?thread_ts=1683056073.574109&cid=C039JSH0807